analysed-packages/less/version-600/less-600-OSS-disclosure.txt

```diff
@@ -0,0 +1,793 @@
+are met:
+
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright
+   notice in the documentation and/or other materials provided with
+   the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
+IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+--------------------------------------------------------------------------------
+
+License-of-GNU-Licenses
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+--------------------------------------------------------------------------------
+
+Spencer-86
+
+Copyright (c) 1986 by University of Toronto. Written by Henry Spencer. Not
+derived from licensed software.
+
+Permission is granted to anyone to use this software for any purpose on any
+computer system, and to redistribute it freely, subject to the following
+restrictions:
+
+1. The author is not responsible for the consequences of use of this software,
+no matter how awful, even if they arise from defects in it.
+
+2. The origin of this software must not be misrepresented, either by explicit
+claim or by omission.
+
+3. Altered versions must be plainly marked as such, and must not be
+misrepresented as being the original software.
+
+Beware that some of this code is subtly aware of the way operator precedence is
+structured in regular expressions. Serious changes in regular-expression syntax
+might require a total rethink.
+
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This configure script is free software; the Free Software Foundation
+gives unlimited permission to copy, distribute and modify it.
+
+This config.status script is free software; the Free Software Foundation
+gives unlimited permission to copy, distribute and modify # it.
+
+--------------------------------------------------------------------------------
+
+Public-domain
+
+Public domain
+
+--------------------------------------------------------------------------------
+
+Dual-license
+
+Dual-license of: Less or GPL-3.0-or-later
+
+--------------------------------------------------------------------------------
+
+X11
+
+X11 License
+
+Copyright (C) 1996 X Consortium
```

Comment:
Can you please reomve the copyright statement here. This is wrong and probably stored in the FOSSology license database as part of the license text. But it is wrong especially here because the copyright sting in the source file says:
Copyright (c) 1987, X Consortium